src/ResourceManager/Compute/Commands.Compute/help/Get-AzureRmGalleryImageVersion.md

```diff
@@ -0,0 +1,147 @@
+---
+external help file: Microsoft.Azure.Commands.Compute.dll-Help.xml
+Module Name: AzureRM.Compute
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/get-azurermgalleryimageversion
+schema: 2.0.0
+---
+
+# Get-AzureRmGalleryImageVersion
+
+## SYNOPSIS
+Get or list gallery image versions.
+
+## SYNTAX
+
+### DefaultParameter (Default)
+```
+Get-AzureRmGalleryImageVersion [-ResourceGroupName] <String> [-GalleryName] <String>
+ -GalleryImageDefinitionName <String> [[-GalleryImageVersionName] <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameter
+```
+Get-AzureRmGalleryImageVersion [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get or list gallery image versions.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzureRmGalleryImageVersion -ResourceGroupName $rgname -GalleryName $gallery -ImageDefinitionName $image -GalleryImageVersionName $version
+```
+
+Get the gallery image version.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -GalleryImageDefinitionName
+The name of the gallery image definition.
+
+```yaml
+Type: System.String
+Parameter Sets: DefaultParameter
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -GalleryImageVersionName
+The name of the gallery image version.
+
+```yaml
+Type: System.String
+Parameter Sets: DefaultParameter
+Aliases: Name
```

Comment:
@hyonholee same comment here: swap the name and alias of this parameter and keep everything else the same